core/tests/fixtures/utils/order/index.ts

```diff
@@ -11,7 +11,7 @@ export class OrderFactory {
   constructor(private readonly page: Page) {}
 
   async create(customerId: number, productId: number) {
-    const orderData = await createOrder(customerId, productId);
+    const orderData = await createOrder(productId, customerId);
```

Comment:
Can we align the method params with this change?

Reply:
thanks, done